neofetch

```diff
@@ -4261,7 +4279,11 @@ info() {
     [[ "$prin" ]] && return
 
     # Update the variable.
-    output="$(trim "${!2:-${!1}}")"
+    if [[ "$2" ]]; then
+        output="$(trim "${!2}")"
+    else
+        output="$(trim "${!1}")"
+    fi
```

Comment:
```sh
output=${!2}
output=$(trim "${output:-${!1}}")
```

Reply:
Oh, that's basically the same thing: trying to use a positional parameter as a variable name when it's unset. So it crashes as well

Reply:
Ah. Alrighty then.